app/Http/Controllers/Rest/CampaignsController.php

```diff
@@ -140,23 +129,19 @@ public function delete(Request $request, $campaign_id)
     public function read(Request $request, $campaign_id)
     {
         // TODO check privileges
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1135)